libs/s25main/WindowManager.cpp

```diff
@@ -356,10 +356,22 @@ void WindowManager::Msg_LeftUp(MouseCoords mc)
 
     // Ggf. Doppelklick untersuche
     unsigned time_now = VIDEODRIVER.GetTickCount();
-    if(time_now - lastLeftClickTime < DOUBLE_CLICK_INTERVAL && mc.GetPos() == lastLeftClickPos)
+
+    // Sehr schwierig auf touch mit dem default dbl-click interval z.B. Mails zu löschen ohne das Fenster zu schließen
+    if(time_now - lastLeftClickTime < (VIDEODRIVER.IsTouch() ? DOUBLE_CLICK_INTERVAL / 3 : DOUBLE_CLICK_INTERVAL))
     {
-        mc.dbl_click = true;
-    } else
+        if(mc.GetPos() == lastLeftClickPos)
+            mc.dbl_click = true;
+        else if(VIDEODRIVER.IsTouch()) // Fast unmöglich 2 mal auf den exakt selben punkt zu tippen
```

Comment:
So if you do hit the same point with touch it is a double click, else it is a right click which closes the window. This sounds surprising